src/main/resources/META-INF/spring.schemas

```diff
@@ -1,5 +1,5 @@
-http\://www.springframework.org/schema/integration/kafka/spring-integration-kafka-3.2.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.2.xsd
-http\://www.springframework.org/schema/integration/kafka/spring-integration-kafka.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.2.xsd
-https\://www.springframework.org/schema/integration/kafka/spring-integration-kafka-3.2.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.2.xsd
-https\://www.springframework.org/schema/integration/kafka/spring-integration-kafka.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.2.xsd
+http\://www.springframework.org/schema/integration/kafka/spring-integration-kafka-3.3.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.3.xsd
+http\://www.springframework.org/schema/integration/kafka/spring-integration-kafka.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.3.xsd
+https\://www.springframework.org/schema/integration/kafka/spring-integration-kafka-3.3.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.3.xsd
+https\://www.springframework.org/schema/integration/kafka/spring-integration-kafka.xsd=org/springframework/integration/kafka/config/spring-integration-kafka-3.3.xsd
```

Comment:
See Spring Integration: we don't support versions for XSD any more.
I think it is time to do the same here as well.

See some `spring.schemas` to see how those existing versions are mapped to our latest versionless in the jar.